asset_dashboard/static/js/components/map_utils/ShowPopup.js

```diff
@@ -10,12 +10,14 @@ export default function ShowPopup({ geojson }) {
         layer.openPopup()
 
         // Make the layer appear selected.
-        layer.setStyle({
-          fillColor: '#3388ff',
-          fillOpacity: '0.2',
-          color: '#3388ff',
-          weight: '4'
-        })
+        if (layer.setStyle) {
```

Comment:
this was causing point types to blow up since they don't have the `setStyle` function